bool is_active
string control_mode

EEpos desired_left_ee_pos
EEpos desired_right_ee_pos
EEpos desired_head_ee_pos

float32MultiArray desired_joint_positions

bool estop
bool ready
bool move
bool stop